tests/lib/rules/max-len.js

```diff
@@ -309,6 +314,17 @@ ruleTester.run("max-len", rule, {
                 }
             ]
         }, {
+            code: "// A comment that exceeds the max comment length, but not the specified max length for code.",
+            options: [40, 4, { comments: 80 }],
```

Comment:
Right, my mistake!  This test should fail only because the comment exceeds the max comment length.  I will change the comment to:

```
// A comment that exceeds the max comment length
```